wallet/responses.go

```diff
@@ -90,6 +90,11 @@ type Transactions struct {
 	Recent []dcrlibwallet.Transaction
 }
 
+// TransactionsWallet is sent in response to Wallet.GetTransactionsByWallet
+type TransactionsWallet struct {
+	Txs []dcrlibwallet.Transaction
+}
+
```

Comment:
Make a struct with all data needed for display (as strings if needed) instead of using `dcrlibwallet.Transaction`